config/routes.rb

```diff
@@ -8,7 +8,15 @@
 
   devise_for :users
 
-  resources :courses, :students
+  resources :courses do
+      post 'courses/:id', to:'courses#show', on: :member
```

Comment:
This will coincide with courses creation

Reply:
Got rid of all this